### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Quantum-Codes/Scratch-Profile-Comments#4] [General]
@sid72020123  hii. Accepted invite?

**Comment by Sid72020123** [comment]
Yes! Thank you!

**Comment by Quantum-Codes** [comment]
Also is this license good? Or some other would have been better?

**Reply by Sid72020123** [reply]
Its good

**Reply by Quantum-Codes** [reply]
:) API complete? Or more?

**Reply by Sid72020123** [reply]
Its complete!

**Reply by Sid72020123** [reply]
I will update scratchconnect when my exams are over

**Comment by Sid72020123** [comment]
I think to unban yourself from scratch you have to appeal to the scratch team

**Reply by Quantum-Codes** [reply]
I appealed multiple times

**Reply by Sid72020123** [reply]
Oh! :( hope you will get unban soon

**Reply by Quantum-Codes** [reply]
It's been 50 days

**Comment by Quantum-Codes** [comment]
Coming in 30 mins.... read issue

**Reply by Sid72020123** [reply]
Ok

**Comment by Quantum-Codes** [comment]
Hi, #5 ..
So this API is slow due to certain reasons..  if u add in ur library, won't it slow down?
Also thx for collaboration

**Reply by Quantum-Codes** [reply]
We can safely delete old repo?

**Reply by Sid72020123** [reply]
Ok

**Reply by Sid72020123** [reply]
As you wish
